app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/background"
+        android:layout_width="fill_parent"
+        android:layout_height="180dp"
+        android:scaleType="centerCrop"
+        android:src="@drawable/login_cover"
+        app:layout_constraintTop_toTopOf="parent"
+        android:foreground="@drawable/gradient"
+        tools:ignore="MissingConstraints" />
+
+    <ImageView
+        android:id="@+id/wolox_image"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/wolox_logo_large"
+        app:layout_constraintBottom_toBottomOf="@id/white"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.899"
+        tools:ignore="MissingConstraints" />
+
+    <View
+        android:id="@+id/white"
+        android:layout_width="match_parent"
+        android:layout_height="60dp"
+        android:layout_marginTop="8dp"
+        android:alpha="0"
+        android:background="#FFFFFF"
+        app:layout_constraintTop_toBottomOf="@id/background"/>
+
+    <EditText
+        android:id="@+id/edittext_first_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:hint="First Name (recomended)"
```

Comment:
Also there is a `strings.xml` that will contain all your string resources.  
More information: https://developer.android.com/guide/topics/resources/string-resource.
Please change it everywhere 🙏